plugins/app-backend/src/index.ts

```diff
@@ -21,4 +21,3 @@
  */
 
 export { appPlugin as default } from './service/appPlugin';
-export * from './service/router';
```

Comment:
This was marked as `@internal` but actually exported - double checking with @Rugvip whether this is actually safe to remove?